style.css

```diff
@@ -79,6 +80,106 @@ button {
   padding-bottom: 12px;
 }
 
+/* popUp window /////////////////////////////////////////// */
+.popup-window.hidden {
+  display: none;
+}
+
+.popup-window {
+  border: 1px solid #dfe1e6;
+  background-color: white;
+  border-radius: 16px;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  row-gap: 12px;
+}
```

Comment:
- [ ] You have done very great to implement the popup modal. However, we need to ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-2702&t=GHreZDvKXP2czKFa-0) (e.g., font, colors, images, text, margins). Therefore I suggest that we change the width and height of the modal and also ensure that the modal is both vertically and horizontally centered to match the Figma template design ☺️ .
- [ ] Kindly ensure that the modal has a blur effect to match the Figma design☺️. This will make our portfolio more professional and have a great UI.

Your desktop design 😎 
![image](https://user-images.githubusercontent.com/99394374/223958994-26ea9105-788c-4e81-a9ea-ed90c5233a64.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/223959087-57b09aba-0ae6-4bcb-88a5-51ff6a99544c.png)

Your mobile design 😎 
![image](https://user-images.githubusercontent.com/99394374/223959368-e1625f83-5d9a-4e07-9957-0f07952c13a1.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/223959468-f077b2b7-f1f3-42c0-b041-0bd86ca66094.png)